telebot/types.py

```diff
@@ -6728,8 +6739,12 @@ def convert_input_media(self):
         """
         if service_utils.is_string(self.media):
             return self.to_json(), None
+        
+        media_dict = {self._media_name: self.media}
+        if self._thumbnail_name:
+            media_dict[self._thumbnail_name] = self.thumbnail
```

Comment:
![image](https://github.com/user-attachments/assets/1485f2df-a3dc-4f02-92bc-b61f004717b1)

media_dict[self._thumbnail_name] will conflict with media_dict[self._media_name] (overrite) if both media and thumbnail is set as "string".

Can we use some value in self._thumbnail_name instead of empty string - I'm not sure, should be tested.